### DISCUSSION TITLE
Authorization on a Resource level

### DISCUSSION THREAD

**Discussion by ghost** [laravel/nova-issues#6462] [Ideas & Feature Requests]
The current [authorization mechanism](https://nova.laravel.com/docs/4.0/resources/authorization.html#policies) requires authorization logic to be defined in Policies. As we know from Laravel, a single Model can have only one Policy. However, one Model can be used in multiple Nova Resources. In such scenarios, it is challenging to implement different authorization logic for each Nova Resource in which the Model is used.

So, it would be highly appreciated if we could define authorization logic on a Resource level, instead of Model -> Policy level.

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
